sklearn/tree/export.py

```diff
@@ -373,8 +375,9 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
     return_string = False
     try:
         if out_file == 'tree.dot':
-            warnings.warn("out_file can be set to None from 0.18, "
-                          "this will be the default in 0.20.", DeprecationWarning)
+            warnings.warn("out_file can be set to None starting from 0.18. "
```

Comment:
Well, if someone wants to keep the old behavior and they have code that uses "tree.dot" I don't think it's unreasonable that someone would pass it.
Have we used the SENTINEL somewhere? I'm slightly concerned about the docstring.... you could do ``SENTINEL.__repr__ = lambda x: "tree.dot" `` [would that work?] ;)